zetaclient/chains/base/signer.go

```diff
@@ -23,6 +23,9 @@ type Signer struct {
 	// logger contains the loggers used by signer
 	logger Logger
 
+	// outboundBeingReported is a map of outbound being reported to tracker
```

Comment:
What the context on this, is it related to Solana?

Reply:
It's related to outbound tracker reporting and being used by both EVM and Solana signers.